api/src/main/java/com/velocitypowered/api/event/ResultedEvent.java

```diff
@@ -72,7 +73,7 @@ public static GenericResult denied() {
         private final boolean allowed;
         private final @Nullable Component reason;
 
-        private ComponentResult(boolean allowed, @Nullable Component reason) {
+        protected ComponentResult(boolean allowed, @Nullable Component reason) {
```

Comment:
This should be made package-private.

Reply:
how i can make this? i can not call super() in PreLoginEvent.PreLoginComponentResult

Reply:
Oof... okay, then keep this protected. I don't know instinctively all the Java access levels off the top of my head!